docs/user_documentation/guide-questionnaire.md

```diff
@@ -37,4 +37,28 @@ Go to the data explorer and look for the (in our case) Simple Questionnaire tabl
 
+# Configure PDF report
+After submitting a questionnaire users can download a PDF report of their data. All questions and answers that were provided will be presented nicely formatted in this report.  
+This PDF report has an optional intro text and logo. To provide these, an extra attribute, with the name "report_header" should be added to the questionnaire. This attribute 
+should be of datatype "xref" with as refEntity a table that specifies the "intro" and "logo". To have the same logo and intro for each participant, use the defaultValue option to
+specify the data row in the config table. Example structure:
  
+Attributes:
+ 
+| name               | entity               | dataType               | idAttribute | labelAttribute | defaultValue | refEntity          | label                                           | partOfAttribute |
+|--------------------|----------------------|------------------------|-------------|----------------|--------------|--------------------|-------------------------------------------------|-----------------|
+| id                 | demo_report_header   | string                 | TRUE        | TRUE           |              |                    |                                                 |                 |
+| intro-en           | demo_report_header   | text                   | FALSE       | FALSE          |              |                    |                                                 |                 |
+| intro-nl           | demo_report_header   | text                   | FALSE       | FALSE          |              |                    |                                                 |                 |
+| intro-fr           | demo_report_header   | text                   | FALSE       | FALSE          |              |                    |                                                 |                 |
+| intro-de           | demo_report_header   | text                   | FALSE       | FALSE          |              |                    |                                                 |                 |
+| intro-es           | demo_report_header   | text                   | FALSE       | FALSE          |              |                    |                                                 |                 |
+| intro-it           | demo_report_header   | text                   | FALSE       | FALSE          |              |                    |                                                 |                 |
+| intro-pt           | demo_report_header   | text                   | FALSE       | FALSE          |              |                    |                                                 |                 |
+| logo               | demo_report_header   | text                   | FALSE       | FALSE          |              |                    |                                                 |                 |
```

Comment:
> The logo should be a data URL (and therefore is of type text)

Attribute type HYPERLINK instead of TEXT for URLs.

Reply:
Looking at the example data I noticed that the logo value is a base64 encoded image. @ConnorStroomberg please add this to the documentation since this is not a intuitive value.